elfpy/agents/agent.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, wallet_address: int, policy: BasePolicy = NoActionPolicy()):
     def __str__(self) -> str:
         # cls arg tells json how to handle numpy objects and nested dataclasses
         dict_to_encode = {str(key): str(value) for key, value in self.__dict__.items()}
```

Comment:
if everything is a string, do we need a custom encoder? :thinking:

Reply:
Actually this is a legacy function from the wallet and would not work. Agent is not a dataclass, so it does not have a `__dict__` function. I removed it completely. We can add a pretty print later if we need it.